nixos/lib/qemu-flags.nix

```diff
@@ -17,9 +17,9 @@ in
         else throw "Unknown QEMU serial device for system '${pkgs.stdenv.hostPlatform.system}'";
 
   qemuBinary = qemuPkg: {
-    x86_64-linux = "${qemuPkg}/bin/qemu-kvm -cpu kvm64";
```

Comment:
This change is causing all of my NixOS tests to fail with error: [Errno 32] Broken pipe when calling machine.shell.send. I am trying to run the tests on a NixOS VM running in Hyper-V. I'm not sure if the nested virtualization is the cause of this.

Reply:
Does `qemu-kvm -cpu host` work or fail with some error?
"Broken pipe" might indicate the qemu process exit.
It might print an useful message before exit

Reply:
It fails with the following error:
```
qemu-system-x86_64: error: failed to set MSR 0x48b to 0x358ae00000000
qemu-system-x86_64: /build/qemu-4.2.0/target/i386/kvm.c:2947: kvm_put_msrs: Assertion `ret == cpu->kvm_msr_buf->nmsrs' failed.
Aborted
```